src/main/java/frc/robot/subsystems/climberSubsystem.java

```diff
@@ -12,14 +12,13 @@
 public class climberSubsystem extends SubsystemBase {
   final LeftClimberSubsystem m_lcs;
   final RightClimberSubsystem m_rcs;
-  final Pigeon2 m_pigeon2;
+  private LimelightNavigation m_ln;
```

Comment:
I would mark this `private final` for the same reasons.

Reply:
Gotchaaaa